src/ui/main.qml

```diff
@@ -348,6 +348,82 @@ Window {
         }
     }
 
+    Connections {
+        target: VPNErrorHandler
+        function onSubscriptionGeneric() {
+            mainStackView.push("qrc:/ui/views/ViewErrorFullScreen.qml", {
+                // Problem confirming subscription...
+                headlineText: VPNl18n.ProblemConfirmingSubscription,
+
+                // An unexpected error has occured, please try again.
+                // Visit our help center below to learn more about how to troubleshoot our VPN.
+                errorMessage: VPNl18n.RestoreUnexpectedError,
```

Comment:
Maybe `VPNl18n.GenericPurchaseErrorGenericPurchaseErrorText`
Does the comment represent what the final text is supposed to be? We have a string that is close to what is in the comment but doesn't match exactly. 
https://github.com/mozilla-mobile/mozilla-vpn-client/blob/0af4fd0606ff975973a2bc1c5c417e9af2d5d180/translations/strings.yaml#L194